src/app/helptext/task-calendar/cloudsync/cloudsync-form.ts

```diff
@@ -32,7 +32,10 @@ encryption_placeholder: T('Server Side Encryption'),
 encryption_tooltip: T('Choose <i>AES-256</i> or <i>None</i>.'),
 
 storage_class_placeholder: T('Storage Class'),
-storage_class_tooltip: T(''),
+storage_class_tooltip: T('See <a\
```

Comment:
Might be helpful to have a brief description of what the storage class is: "Classification for each S3 object. Choose a class based on the specific use case or performance requirements." (adapted from Amazon docs)